docs/topics/Threads.md

```diff
@@ -161,3 +161,32 @@ Additionally, when a client loses access to a channel they are not removed from
+
+# Forums
+
+A `GUILD_FORUM` channel is very similar (from an API perspective) to a `GUILD_TEXT` channel, except only threads can be created in that channel; messages cannot be sent directly in that channel.
+
+This feature is still in active development. Many aspects are in flux and subject to change, including (but not limited to):
+- The name of the feature
+- The permissions required to create threads in a forum (currently `SEND_MESSAGES` but may change)
+- The shape of data exposed on the channel objects
+
+> warn
+> Documentation for those features will be published in the future, once they have been finalized. We recommend developers avoid implementing features that are undocumented or are documented as still in flux.
+
+Listed below are some of the important details of forum channels and are safe to implement. Many bots can have reasonable support for the new channel simply by making it aware of the new channel type so it starts processing gateway events for that channel (and thus unlocking things like bots used for moderation):
+
+- Forums are a new channel type `15`.
+- Forums do not allow messages to be sent in them. Endpoints like /channels/channel_id/messages will not work on a forum channel.
+- Threads can be created in a forum channel. All threads in a forum are of type `GUILD_PUBLIC_THREAD`. These threads and messages within the thread have the same gateway events as threads in a normal text channel.
+- The APIs for loading active & archived threads, joining & leaving a thread, and loading who is in a thread are unchanged and work the same for threads in a forum.
+- The API to create a thread in a forum will create _both_ a thread and message in the same call, and as such requires passing in parameters for both the thread and message. The name and behavior of parameters is the same as they are for the existing create thread/message endpoints to simplify integrating with it.
+- The message created by that API call will have the same id as the thread.
+
+Listed below are things that are unlikely to change, but still might:
+
+- The `topic` field on a forum channel is what is shown in the "Guidelines" section visually
+- The `rate_limit_per_user` field currently behaves the same as in a text channel, limiting how frequently threads can be created, _but its behavior may be changed before launching_.
+- Threads in a forum have the same permissions behavior as threads in a text channel, inheriting all permissions from the parent channel, with just one exception: Creating a thread in a forum channel only requires the permission that is currently named `SEND_MESSAGES`.
+- The first message in a forum channel can contain additional markdown for bulleted list and headings.
```

Comment:
The wording here is not clear here. Is it supposed to say 'first message in threads created in a forum channel'?

Reply:
Yeah it's referring to a thread in a forum channel, you cannot send messages in the forum channel directly.